tests/marginModels/test_pointSet_margin_model.xml

```diff
@@ -0,0 +1,80 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>plugins/SR2ML.marginModel.PointSetMarginModel</name>
+    <author>mandd</author>
+    <created>2021-06-07</created>
+    <classesTested>Models.ExternalModel</classesTested>
+    <description>
+      This input tests the PointSetMarginModel model 1D data.
+      The failure data was generated from a uniform distribution [9,10] (i.e., =9.0+RAND() ).
+      Given actualTime between 2 and 3, the normalize margin should be distributed between. 
+    </description>
+
+  </TestInfo>
+  
+  <RunInfo>
+    <WorkingDir>PointSetModel</WorkingDir>
+    <Sequence>MCRun,print</Sequence>
+  </RunInfo>
+  
+  <Models>
+    <ExternalModel name="PointSetMargin" subType="SR2ML.MarginModel">
+      <variables>actualTime,marginPS1</variables>
+      <MarginModel type="PointSetMarginModel">
+        <failedDataFileID>failureData.csv</failedDataFileID>
+        <marginID>marginPS1</marginID>
+	      <map var='time'>actualTime</map>
+      </MarginModel>
+    </ExternalModel>
+  </Models>
+
+  <Distributions>
+    <Uniform name="actualTimeDist">
+      <lowerBound>1.999999</lowerBound>
+      <upperBound>2.000001</upperBound>
+    </Uniform>
+  </Distributions>
+
+  <Samplers>
+    <MonteCarlo name="MC">
+      <samplerInit>
+        <limit>1000</limit>
```

Comment:
can you reduce the number of sampling?

Reply:
If possibly, I'd like to keep large since it is an analytical test.
I am planning to extend this test by adding a basic statistics PP in the future